app/bundles/LeadBundle/Translations/en_US/messages.ini

```diff
@@ -210,10 +210,13 @@ mautic.config.tab.companyconfig="Company Settings"
 mautic.config.tab.importconfig="Import Settings"
 mautic.config.tab.contact.list.settings="Contact List Settings"
 mautic.config.tab.columns="Columns"
+mautic.config.tab.contact.settings="Contact Settings"
 mautic.config.tab.contact.merge.settings="Contact Merge Settings"
 mautic.config.tab.company.merge.settings="Company Merge Settings"
 mautic.core.config.unique_identifiers_operator="Merge by unique fields with operator"
 mautic.core.config.unique_identifiers_operator.tooltip="If you use more than one unique fields identifiers then you can define the way of the find to merge it. "
+mautic.core.config.allow_multiple_companies="Multiple companies for contact"
+mautic.core.config.allow_multiple_companies.tooltip="Enabled option allow assign to contact multiple companies."
```

Comment:
Could we start to apply our styleguide to these messages by capitalising the Contact and Company when mentioning them in the context of the 'thing' in Mautic? It would help us to improve consistency with our documentation which already has these rules in place.